README.md

```diff
@@ -16,7 +26,23 @@ An website for watching videos together.
 - Text chat
 - Video chat
 
-## Configuration
+## Advanced Setup
+
+### YouTube API
+
+This project is using the YouTube API which means you will need to setup an API key. You can get one from Google [here](https://console.developers.google.com).
```

Comment:
Might want to mention this is actually optional, if YouTube isn't required

Reply:
True!